### DISCUSSION TITLE
Created a benchmark of generating JSON Schema

### DISCUSSION THREAD

**Discussion by ryoppippi** [samchon/typia#1207] [Show and tell]
I created a small benchmark for generating JSON Schema with
- typia
- zod
- typebox
- tschema

and typia always wins!!

Repo: https://github.com/ryoppippi/JSON-Schema-Benchmark
<img width="873" alt="Screenshot 2024-08-07 at 19 32 32" src="https://github.com/user-attachments/assets/96a84b7b-e82d-429d-bb66-413ee695b039">

**Comment by samchon** [comment]
@{number=1207; body=Thanks for sharing the benchmark. This is useful community evidence, but it does not require a repository change here by itself.

Closing this show-and-tell discussion as acknowledged.}.body
